### PR TITLE
wazo_install.sh: set the upgrade dist and repo

### PR DESCRIPTION
**Pull request by fredericlepied**

### CODE REVIEW

File: wazo_install.sh [hunk 1]
```diff
@@ -113,7 +113,9 @@ install_wazo_with_ansible() {
     # Run Ansible
     echo "Installing $tag" 1>&1
     /var/lib/wazo-ansible-venv/bin/ansible-galaxy install -r requirements-postgresql.yml
-    /var/lib/wazo-ansible-venv/bin/ansible-playbook -i inventories/uc-engine --extra-vars "{\"wazo_distribution\":\"$distribution\", \"wazo_debian_repo\": \"$repo\"}" uc-engine.yml
+    /var/lib/wazo-ansible-venv/bin/ansible-playbook -i inventories/uc-engine \
+        --extra-vars "{\"wazo_distribution\": \"$distribution\", \"wazo_debian_repo\": \"$repo\", \"wazo_debian_repo_upgrade\": \"$upgrade_repo\", \"wazo_distribution_upgrade\": \"$upgrade_dist\"}" \
```
Comment (sduthil) [inline wazo_install.sh]:
```suggestion
        --extra-vars wazo_distribution="$distribution"\
        --extra-vars wazo_debian_repo="$repo" \
        --extra-vars wazo_debian_repo_upgrade="$upgrade_repo" \
        --extra-vars wazo_distribution_upgrade="$upgrade_dist \
```

**Comment by sduthil** [comment]
The upgrade logic goes like:
* install from `phoenix-stretch` upgrades to `phoenix-stretch` (further upgrades to `pelican-stretch` will be handled by `wazo-upgrade`)
* install from `pelican-stretch` upgrades to `pelican-stretch`
* install from `wazo-dev-stretch` upgrades to `wazo-dev-stretch`
* install from `wazo-rc-stretch` upgrades `wazo-rc-stretch`
* install from archive versions after `wazo-19.10` upgrade to `pelican-stretch`
* install from archive versions before `wazo-19.10` upgrade to `phoenix-stretch`
@fblackburn1 do you agree?

**Comment by fredericlepied** [comment]
@sduthil LGTM. Could you merge it?

**Comment by fblackburn1** [comment]
> The upgrade logic goes like:
>     * install from archive versions after `wazo-19.10` upgrade to `pelican-stretch`
> 
>     * install from archive versions before `wazo-19.10` upgrade to `phoenix-stretch`
>       @fblackburn1 do you agree?

Note that `pelican-stretch` start from `wazo-19.04` not `wazo-19.10`
Since the first step of wazo-upgrade is to update wazo-upgrade:
 * install from archive versions before `wazo-19.04` upgrade to `phoenix-stretch` AND the upgrade procedure will switch to `pelican-stretch`.

Since `wazo-upgrade` must be identical from `phoenix-stretch` and `pelican-stretch`, that make no difference what is the value of `upgrade_distribution`

**Pull request merged**